invenio_communities/views/communities.py

```diff
@@ -267,6 +268,11 @@ def load_custom_fields(dump_only_required=False):
 @login_required
 def communities_new():
     """Communities creation page."""
+    warn(
```

Comment:
minor: shouldn't this only be logged if we know for sure the `/communities/new` endpoint is being used? Otherwise it will show up always, even when folks access the new `/communities-new` endpoint.

Reply:
Yes ideally that should be the case, but having a general warning should be fine since we're gonna remove it in the future anyways?
If we want it only for the deprecated, would it make sense to have a generic wrapper?

Reply:
My take on warnings is that someone should be able to do something about to make them go away. Imagine e.g. folks that use the `-W` flag in pytest, they wouldn't have any way to address the warning (besides explicitly ignoring it in test config).

In this case it's just going to be noise in the terminal everytime you run the application and access the page (even with the correct URL).

Having a new view function (using a wrapper or similar) and registering would be the way to go IMO.